apis/reaper/v1alpha1/reaper_types.go

```diff
@@ -212,6 +213,11 @@ type ReaperSpec struct {
 	// +kubebuilder:default="LOCAL"
 	// +kubebuilder:validation:Enum:=LOCAL;ALL;EACH
 	DatacenterAvailability string `json:"datacenterAvailability,omitempty"`
+
+	// Client encryption stores which are used by Cassandra and Reaper.
+	// These should not be set explicitly by the user, they will be copied from the K8ssandraCluster object.
```

Comment:
Same remark about this comment: better remove it as it doesn't concern standalone Reapers.

Reply:
👍